src/bots/modchat/messageHandler.js

```diff
@@ -0,0 +1,50 @@
+import { DMChannel } from 'discord.js';
+import client from '../../client';
+import generateEmbed from '../../utils/embed';
+import { getStrings } from './constants';
+import { v4 as uuidv4 } from 'uuid';
+import { getModChannel, parseCommandString } from '../../utils/index';
+
+/**
+ * Handles direct messages from users to forward to moderation team.
+ */
+let handlePrivateMessage = async () => {
+  let cmd = parseCommandString();
+  let channel = client.message.channel;
+
+  if (cmd.arguments.length === 0) {
+    channel.send(getStrings().explanation);
```

Comment:
maybe we shouldn't call `getStrings()` repeatedly if there are no options passed through, just call it once at the top of the handler and store returned object in a variable